srcpkgs/monero-gui/template

```diff
@@ -1,18 +1,18 @@
 # Template file for 'monero-gui'
 pkgname=monero-gui
```

Comment:
can we add a comment to both packages to keep them in sync?

Reply:
(releases aren't always in sync - see other comment)